security/pkg/nodeagent/caclient/client.go

```diff
@@ -48,6 +50,9 @@ func NewCAClient(endpoint, CAProviderName string, tlsFlag bool) (caClientInterfa
 	switch CAProviderName {
 	case googleCAName:
 		return gca.NewGoogleCAClient(endpoint, tlsFlag)
+	case vaultCAName1:
+		//TO-DO (lei-tang): fill vault parameters from environmental variables
+		return vault.NewVaultClient(tlsFlag, nil, "vaultAddr", "vaultLoginRole", "vaultLoginPath", "vaultSignCsrPath")
```

Comment:
is ubuntu CA bundle is good enough to call vault, or do you plan to pass some other tlsRootCert here ?